ibis/backends/dask/tests/execution/conftest.py

```diff
@@ -71,7 +71,7 @@ def df():
             'map_of_complex_values': [None, {'a': [1, 2, 3], 'b': []}, {}],
         }
     )
-    return dd.from_pandas(pandas_df, npartitions=1)
+    return dd.from_pandas(pandas_df, npartitions=2)
```

Comment:
Probably better to define a variable for default num partitions in case we want to change this again.